src/containers/RecruiterDashboard.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { Button, Link, Box, Grid, makeStyles } from "@material-ui/core";
+
+
+import { StatsComponent, SummaryComponent } from "../common";
+import oracle from "../assets/svg/oracle.svg";
+
+import { Navbar, Footer } from "../common";
+
+
+const recruiterData = {
+    applied: {
+        title: 'Jobs Posted', 
+        value: 36
+    }, 
+    saved: {
+        title: 'Hired', 
+        value: 14
+    }, 
+    viewed: {
+        title: 'In Progress', 
+        value: 22
+    },
+    invitations: {
+        title: 'Scheduled', 
+        value: 5
+    }
+}
+
+    
+
+const jobPostData = {
+    type: 'companyDetails', 
+    companyLogo: oracle,
+    companyName: 'oracle', 
+    jobTitle: 'Frontend Developer', 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
    jobTitle: "Frontend Developer", 
```